test/run-qemu

```diff
@@ -46,4 +46,5 @@ if [[ $* == *-initrd* ]]; then
     ARGS+=(-kernel "$VMLINUZ")
 fi
 
+echo "${0##*/}: $BIN ${ARGS[*]@Q} ${*@Q}"
```

Comment:
This line is ugly to read, but since there are parameters that contain spaces (e.g. `-append`), this is the only way I found to do it in one line. The alternative would be a five line long for loop.